src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/MetricItemExtensions.cs

```diff
@@ -97,11 +97,21 @@ internal static OtlpMetrics.ScopeMetrics GetMetricListFromPool(string name, stri
                     Version = version ?? string.Empty, // NRE throw by proto
                 },
             };
+
+            if (meterTags != null)
+            {
+                AddAttributes(meterTags, metrics.Scope.Attributes);
+            }
         }
         else
         {
             metrics.Scope.Name = name;
             metrics.Scope.Version = version ?? string.Empty;
+            if (meterTags != null)
+            {
+                metrics.Scope.Attributes.Clear();
```

Comment:
I think we should clear these attributes in the `Return` method to stay consistent with how we clear `ScopeMetrics.Metrics`.

https://github.com/open-telemetry/opentelemetry-dotnet/blob/324ab61fa190880222468992f29844df2559ac24/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/MetricItemExtensions.cs#L72-L85

Reply:
Thanks. https://github.com/open-telemetry/opentelemetry-dotnet/pull/5089#discussion_r1408254794 is suggesting the same thing I believe.